app/resources/game_content.py

```diff
@@ -0,0 +1,203 @@
+import math
+from typing import Tuple, List
+
+from app import m
+from models.hardware import Hardware
+from models.service import Service
+from models.workload import Workload
+from vars import hardware, resolve_ram_type, resolve_gpu_type
+
+
+def check_exists(user: str, elements: dict) -> Tuple[bool, dict]:
+    response: dict = m.contact_microservice("inventory", ["inventory", "list"], {"owner": user})
+
+    names: List[str] = [x["element_name"] for x in response["elements"]]
+
+    if elements["cpu"] not in names:
+        return False, {"error": "you_dont_own_such_cpu"}
+    if elements["motherboard"] not in names:
+        return False, {"error": "you_dont_own_such_motherboard"}
+    if elements["gpu"] not in names:
+        return False, {"error": "you_dont_own_such_gpu"}
+    for ram in elements["ram"]:
+        if ram not in names:
+            return False, {"error": "you_dont_own_such_ram"}
+        else:
+            names.remove(ram)
+    for disk in elements["disk"]:
+        if disk not in names:
+            return False, {"error": "you_dont_own_such_disk"}
+        else:
+            names.remove(disk)
+
+    return True, {}
+
+
+def delete(user: str, elements: dict):
+    m.contact_microservice("inventory", ["inventory", "delete_by_name"], {"owner": user, "item_name": elements["cpu"]})
+    m.contact_microservice(
+        "inventory", ["inventory", "delete_by_name"], {"owner": user, "item_name": elements["motherboard"]}
+    )
+    m.contact_microservice("inventory", ["inventory", "delete_by_name"], {"owner": user, "item_name": elements["gpu"]})
+    for ram in elements["ram"]:
+        m.contact_microservice("inventory", ["inventory", "delete_by_name"], {"owner": user, "item_name": ram})
+    for disk in elements["disk"]:
+        m.contact_microservice("inventory", ["inventory", "delete_by_name"], {"owner": user, "item_name": disk})
+
+
+def check_elements_exist(elements: dict) -> Tuple[bool, dict]:
+    if not elements["cpu"] in hardware["cpu"].key():
+        return False, {"error": "element_cpu_not_found"}
+    if not elements["gpu"] in hardware["gpu"].keys():
```

Comment:
`hardware["gpu"]` does not exist
```suggestion
    if elements["gpu"] not in hardware["graphiccards"]:
```